kf_lib_data_ingest/templates/my_study/transform_module.py

```diff
@@ -8,11 +8,12 @@
 implementing transform_function.
 """
 
-from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.common.concept_schema import CONCEPT  # noqa F401
```

Comment:
Why do we want some lines not to be checked by the linter? It seem neither exceeding the character limit nor break linting rules?